catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/transformer/TransformerDescriptors.java

```diff
@@ -33,36 +38,58 @@ public TransformerDescriptors(
   }
 
   public List<Map<String, String>> getMetacardTransformers() {
-    return getTransformerDescriptors(metacardTransformers);
+    return getTransformerDescriptors(metacardTransformers, blackListedMetacardTransformers);
   }
 
   public List<Map<String, String>> getQueryResponseTransformers() {
-    return getTransformerDescriptors(queryResponseTransformers);
+    return getTransformerDescriptors(
+        queryResponseTransformers, blackListedQueryResponseTransformers);
   }
 
   public Map<String, String> getMetacardTransformer(String id) {
-    return getTransformerDescriptor(metacardTransformers, id);
+    return getTransformerDescriptor(metacardTransformers, blackListedMetacardTransformers, id);
   }
 
   public Map<String, String> getQueryResponseTransformer(String id) {
-    return getTransformerDescriptor(queryResponseTransformers, id);
+    return getTransformerDescriptor(
+        queryResponseTransformers, blackListedQueryResponseTransformers, id);
+  }
+
+  public List<String> getBlackListedMetacardTransformers() {
+    return blackListedMetacardTransformers;
+  }
+
+  public List<String> getBlackListedQueryResponseTransformers() {
+    return blackListedQueryResponseTransformers;
+  }
```

Comment:
✏️ Can this method be removed because the `blackListedQueryResponseTransformers` are only used in the `getQueryResponseTransformers` and `getMetacardTransformer` public methods?

Reply:
This getter can be removed. I am exposing `getBlackListedMetacardTransformerIds` and using this method elsewhere which could be used as a reason to leave it. However I will remove it for now since adding a getter is trivial.